src/index.js

```diff
@@ -28,3 +28,4 @@ export BrowserDetector from './browser-detector';
 export TimePicker from './time-picker';
 export FocusableView from './focusable-view';
 export ResizeSensor from './resize-sensor';
+export Tablo, { Column as TabloColumn, Cell, Header, Footer } from './tablo';
```

Comment:
I had to rename `Column` to `TabloColumn` to avoid name collision with `Table` component.
Not so nice, as example could be confusing (`import { TabloColumn } from '...'` won't work)